cypress/e2e/rooms/changeRoomsPermission/changeRoomPermissionViewerToAdmin.feature

```diff
@@ -0,0 +1,99 @@
+@regression_test
+@stable_test
+Feature: Room - Change room permission (Viewer - Admin)
+
+    As a teacher, I want to change a participant’s room permission from viewer to admin, so that other users can access the room and leave it as needed
+
+    Scenario: Teacher change room permission of other participants, including pre-conditions
+        Given I am logged in as a '<teacher_2>' at '<namespace>'
+        Given I am logged in as a '<teacher_1>' at '<namespace>'
+
+        # pre-condition: first teacher creating a new room
+        When I go to room overview
+        When I click on FAB to create new room
+        Then I see room creation page
+        When I enter the room name '<room_name>'
+        When I click on the button to save the room
+        Then I see the detail page of room '<room_name>'
+
+        # first teacher is able to add participants
+        Then I see the detail page of room '<room_name>'
+        When I click on three dot menu in room page
+        When I select the three dot menu action 'room-members'
+        Then I see the edit participants page of room '<room_name>'
+        When I click on FAB to add participants
+        Then I see add participants modal
+        Then I see school '<school_name>' in school dropdown
+        Then I see role '<role_name>' in role dropdown
+        When I enter '<participant_name>' in name dropdown
+        When I select the first name from the dropdown
+        When I click on the button to add the participant
+        Then I see '<participant_name>' in the room participants list
+
+        # first teacher changes role for other user
+        When I click on button Three Dot Menu to add participant '<participant_name>'
+        Then I see button Change Role Permission is visible
+        When I click on button 'Change-Permission' in the sub-menu
+        Then I see dialog box Change Role Permission is visible
+        When I change second user role to 'admin'
+        Then I see Role changed to 'admin' for second user
+        Then I click on button 'Confirm' in the action menu
+
+        # second user logs in, accesses the room, verifies that restricted functionalities are not visible to them, and then leaves the room
+        Given I am logged in as a '<teacher_2>' at '<namespace>'
+        When I go to room overview
+        Then I see '<room_name>' on room overview page
+        When I go to room '<room_name>'
+        Then I see the detail page of room '<room_name>'
+        Then I see button Fab Create Room Board
+        When I click on three dot menu in room page
+        Then I don't see 'delete' options in the menu
+        When I select the three dot menu action 'edit'
+        When I enter the room name '<room_name_edited>'
+        When I click on the button to save the room
```

Comment:
button Save Room (or whatever it is called

Reply:
done